### PR TITLE
Assign bandwidth limit on client VPN connections

### PR DESCRIPTION
**Pull request by ababo**
This is an initial implementation of traffic control for the client VPN sessions. Currently only download rate is constrained and on Linux only.

This commit includes the following changes:
1. The start session call now returns an offering.
2. Added a new `tc` package for traffic control.
3. Dappvpn calls SetRateLimit/UnsetRateLimit.

Resolves BV-786.

### CODE REVIEW

File: sesssrv/start.go [hunk 1]
```diff
@@ -36,6 +41,13 @@ func (s *Server) handleStart(
 		return
 	}
 
+	var offer data.Offering
+	if err := s.db.FindByPrimaryKeyTo(&offer, ch.Offering); err != nil {
+		logger.Error(err.Error())
+		s.RespondError(logger, w, srv.ErrInternalServerError)
```
Comment (furkhat) [inline sesssrv/start.go]: What if err = `sql.ErrNoRows`?

File: tc/tc_linux.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	return nil
+}
+
+func classID(ip net.IP) string {
+	return fmt.Sprintf("1:%x", uint16(crc32.ChecksumIEEE(ip)))
+}
+
+func withMask(ip net.IP) string {
+	return fmt.Sprintf("%s/%d", ip, len(ip))
+}
```
Comment (furkhat) [inline tc/tc_linux.go]: WOW, this file needs more comments for sure.

File: sesssrv/start.go (hunk 1 above)
Comment (ababo) [reply sesssrv/start.go]: It will be an internal database inconsistency error.

File: tc/tc_linux.go (hunk 2 above)
Comment (ababo) [reply tc/tc_linux.go]: I'll put some link to a nice manual for linux traffic control: no need to duplicate the info existing in numerous web documents.

**Pull request merged**